### PR TITLE
Closes #1482 : Add a little padding on top of the allergen alert

### LINKED ISSUE #1482: Add a little padding on top of the allergen alert
image

### PR DESCRIPTION
**Pull request by jaindiv26**
## Description

Made padding equal (Left, Right, Top & Bottom).

## Related issues and discussion
Add a little padding on top of the allergen alert

Issue Number: #1482 
 
 ## Screen-shots, if any
 
screenshot_1525972756

### CODE REVIEW

**Comment by huzaifaiftikhar** [comment]
@jaindiv26 I see that someone else was working on this issue. Please ask the person if he/she is still working on the issue before creating a pull request. 

Padding was required for the entire warning card. The card should be given slight padding at the top so that it doesn't stick at the top as it is currently.

File: app/src/main/res/layout/fragment_summary_product.xml [hunk 1]
```diff
@@ -129,7 +129,10 @@
                                 android:layout_toRightOf="@+id/message_icon"
                                 android:layout_toStartOf="@+id/message_dismiss_icon"
                                 android:gravity="center"
-                                android:paddingRight="@dimen/padding_normal"
+                                android:paddingRight="24dp"
+                                android:paddingTop="4dp"
+                                android:paddingBottom="4dp"
+                                android:paddingLeft="4dp"
```
Comment (huzaifaiftikhar) [inline app/src/main/res/layout/fragment_summary_product.xml]: Add padding to the top of the CardView not the TextView

Comment (jaindiv26) [reply app/src/main/res/layout/fragment_summary_product.xml]: @huzaifaiftikhar Please review my changes, uploading a screenshot of changes done.

screenshot_1526046696

Comment (huzaifaiftikhar) [reply app/src/main/res/layout/fragment_summary_product.xml]: Good to merge once the travis build completes!

**Comment by huzaifaiftikhar** [comment]
Merged, thanks @jaindiv26

**Pull request merged**